config/initializers/gem-plugin_config/resque_mailer.rb

```diff
@@ -1 +1,10 @@
 Resque::Mailer.excluded_environments = [:test, :cucumber]
+Resque::Mailer.error_handler = lambda { |mailer, message, error, action, args|
+  if error.is_a?(Resque::TermException)
+    Resque.enqueue(mailer, action, *args)
+  elsif error.is_a?(ActiveRecord::RecordNotFound) && !args.include?(:retried)
+    Resque.enqueue_in(5.minutes, mailer, action, :retried, *args)
```

Comment:
This looks like it prepends the symbol `:retried` to the arguments passed to the mailer. Doesn't that mean all the arguments will be shifted over by 1, and the first argument (which is frequently supposed to be an ID of some sort) will be `:retried`? Or is there some code that strips the `:retried` symbol?

Also, I poked around the mailer code and I think `Resque::Mailer.perform` expects the arguments to be serialized:
https://github.com/zapnap/resque_mailer/blob/35c2a7c7ff9038b9bff95c50190d162f95afcd5f/lib/resque_mailer.rb#L52-L54